helix-event/src/redraw.rs

```diff
@@ -0,0 +1,51 @@
+//! Signals that control when/if the editor redraws
+
+use std::future::Future;
+
+use parking_lot::{RwLock, RwLockReadGuard};
+use tokio::sync::Notify;
+
+/// A `Notify` instance that can be used to (asyronously) request
```

Comment:
```suggestion
/// A `Notify` instance that can be used to (asynchronously) request
```

Reply:
I forced pushed this since I was fixing it anyway